api/_bot/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Added bot-level `.babelrc` so `babel-watch` will generate code for node instead code for the browser

Reply:
🎉